lib/stack_master.rb

```diff
@@ -54,6 +54,18 @@
 module StackMaster
   extend self
 
+  def interactive?
+    !non_interactive?
```

Comment:
It may be worthwhile to attempt to check if stdout is a tty.  So:

`!non_interactive? && $stdout.tty?`